app/api/helpers/scheduled_jobs.py

```diff
@@ -2,14 +2,14 @@
 from app.models.order import Order
 from app.models.event_invoice import EventInvoice
 from app.models.ticket import Ticket
+from app.models.ticket_fee import get_fee
```

Comment:
What's the need to import it here?

Reply:
`get_fee` is used as a scheduled job to send out the fees notification so this shares the same calculation